samples/net/https_client/sample.yaml

```diff
@@ -1,13 +1,22 @@
 sample:
   name: HTTPS client sample
 tests:
-  sample.cellular.https_client:
+  sample.net.https_client.lte:
     build_only: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
       - nrf9161dk_nrf9161_ns
     platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
     tags: ci_build
+  sample.net.https_client.wifi:
+    build_only: true
+    integration_platforms:
+      - nrf7002dk_nrf5340_cpuapp
+    platform_allow: nrf7002dk_nrf5340_cpuapp
+    extra_configs:
+      - CONFIG_WIFI_CREDENTIALS_STATIC_SSID="test-ssid"
+      - CONFIG_WIFI_CREDENTIALS_STATIC_PASSWORD="test-password"
+    tags: ci_build
   sample.cellular.https_client.tfm-mbedtls:
     build_only: true
     extra_args: OVERLAY_CONFIG="overlay-tfm_mbedtls.conf"
```

Comment:
I think we need another build configuration that builds with this overlay for wifi

Reply:
This is not needed, it does not make sense to build the sample with this overlay for wifi, as it is already using Mbed TLS.